src/rust/protover/ffi.rs

```diff
@@ -105,9 +105,10 @@ pub extern "C" fn protocol_list_supports_protocol(
         Ok(n) => n.into(),
         Err(_) => return 0,
     };
-    match proto_entry.supports_protocol(&protocol, &version) {
-        false => return 0,
-        true  => return 1,
+    if proto_entry.supports_protocol(&protocol, &version) {
+        1
+    } else {
+        0
```

Comment:
Alternatively, maybe putting the if-else in a single line as in https://github.com/rust-lang-nursery/fmt-rfcs/blob/master/guide/expressions.md#single-line-if-else
might be good.

Reply:
Considering `rustfmt` didn't automatically put it on one line, I assume it's not considered "small", as per this from your link:

> ...it contains a single else clause, and is *small*.

Reply:
I think it's in expression context, because it ends up being the implicitly returned value, so the single line if-else is allowed. It's not part of a larger statement or expression on the same line though, so maybe it's in a gray area, especially given the counterexample later in that section. (There's not enough context to tell if the multiline if-else is the last expression of a larger block.)